### PR TITLE
AWS cleanup: Enable treatment of different prefixes with different max running hours

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
@ebenahar how about 72 hours? tier4a is long running suite

**Comment by ebenahar** [comment]
[cleanup_log.zip](https://github.com/red-hat-storage/ocs-ci/files/4216393/cleanup_log.zip)

File: ocs_ci/cleanup/aws/cleanup.py [hunk 1]
```diff
@@ -47,7 +47,9 @@ def get_clusters_to_delete(time_to_delete, region_name, prefixes_to_spare):
         time_to_delete (int): The maximum time in seconds that is allowed
             for clusters to continue running
         region_name (str): The name of the AWS region to delete the resources from
-        prefixes_to_spare (list): The cluster prefixes to spare
+        prefixes_hours_to_spare (list): Dictionaries of the cluster prefixes to spare
```
Comment (vasukulkarni) [inline ocs_ci/cleanup/aws/cleanup.py]: Nice!

Comment (vavuthu) [reply ocs_ci/cleanup/aws/cleanup.py]: s/list/dict/

Comment (ebenahar) [reply ocs_ci/cleanup/aws/cleanup.py]: fixed

**Pull request merged**